examples/application.rs

```diff
@@ -600,8 +605,8 @@ impl ApplicationHandlerExtMacOS for Application {
 
 /// State of the window.
 struct WindowState {
-    /// IME input.
-    ime: bool,
+    /// State of the text input exposed to IME
+    ime_state: Option<ImeState>,
```

Comment:
Can keep it as a bool.

Reply:
Same as above, the entire state must be cached.